plugin/plugin_test.go

```diff
@@ -203,8 +202,8 @@ func TestDistributedSystemsSupport(t *testing.T) {
 		if _, err := clientSvr.Client().Get(fmt.Sprintf("%v?id=1", svr3.URL)); err != nil {
 			log.Println(err)
 		}
+		time.Sleep(1 * time.Second)
```

Comment:
Could you elaborate the reason for this?

Reply:
There is no reason for this.
I have just simulated a test case, where I am sending 1 req/sec to check how the distributed ratelimit is working. If I send 100reqs right now I won't be able to see how things are going on step by step.

Although I can simulate a test case with 100req/second but that will require me to make it like a proper test case and for that as I asked in slack we will have to decide in the config part.